package.json

```diff
@@ -21,8 +22,11 @@
     "orbit-db": "^0.19.8",
     "origin": "~0.7.0",
     "popper.js": "^1.14.3",
+    "query-string": "6.1.0",
+    "rc-slider": "8.6.1",
     "react": "^16.3.2",
     "react-avatar-editor": "^11.0.2",
+    "react-dates": "17.2.0",
     "react-dom": "^16.3.2",
```

Comment:
These dependencies all look good. I'll take another pass at the entire PR once the conflicts are resolved.